DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -74,7 +74,14 @@ public function load(array $configs, ContainerBuilder $container)
             foreach ($config['resolve_target_documents'] as $name => $implementation) {
                 $def->addMethodCall('addResolveTargetDocument', [$name, $implementation, []]);
             }
-            $def->addTag('doctrine_mongodb.odm.event_subscriber');
+
+            // Register service has an event subscriber if implement interface
```

Comment:
@alcaeus I can only end up doing this to check if service class implements interface. Do you think of a better checking code?

Reply:
you should not trim the `%`. If any parameter is involved, it should be resolved, not replaced by its name.